exist-core/src/main/java/org/exist/xquery/functions/xmldb/FunXCollection.java

```diff
@@ -21,37 +21,48 @@
  */
 package org.exist.xquery.functions.xmldb;
 
-//import org.apache.logging.log4j.LogManager;
-
-import org.exist.dom.QName;
-import org.exist.xquery.Cardinality;
 import org.exist.xquery.FunctionSignature;
+import org.exist.xquery.XPathException;
 import org.exist.xquery.XQueryContext;
 import org.exist.xquery.functions.fn.ExtCollection;
-import org.exist.xquery.value.FunctionReturnSequenceType;
-import org.exist.xquery.value.FunctionParameterSequenceType;
-import org.exist.xquery.value.SequenceType;
+import org.exist.xquery.value.Sequence;
 import org.exist.xquery.value.Type;
 
+import java.net.URI;
+
+import static org.exist.xquery.FunctionDSL.*;
+import static org.exist.xquery.functions.xmldb.XMLDBModule.functionSignature;
+
 /**
- * @author wolf
- * @author ljo
```

Comment:
I'd say we either remove author tags altogether or correct them where found. Just removing is against the rules of conduct.

Reply:
I removed them, because this is not the same file. I wrote this from scratch. So it is not a derivative of the original work, so it is fine.

Reply:
However, that said, if you want to remove all the `@author` declarations from source files in eXist-db then I would support that, as I don't generally regard them as a good thing.

Reply:
I am in favour of no authors, but I also think this should be discussed briefly in the next call on Monday so that more people have a chance to react.

Reply:
No decision was made on this issue in the previous two calls.

Reply:
As agreed in today's Community Call, contributors will henceforth remove author tags when they touch a file in a PR. Please remember to remove author tags in your PRs.